src/app/containers/IndexHeading/index.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin } from '@bbc/gel-foundations/scripts';
+import IndexHeading from '.';
+
+describe('Index Heading', () => {
+  describe('snapshot', () => {
+    shouldMatchSnapshot(
+      'should render correctly',
+      <IndexHeading script={latin} service="news">
```

Comment:
Same here, regarding the example
```suggestion
      <IndexHeading script={cyrillicAndLatin} service="ukrainian">
```